lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -142,7 +142,10 @@ export class ManagedIdentityApplication {
             correlationId: this.cryptoProvider.createNewGuid(),
         };
 
-        if (managedIdentityRequest.forceRefresh) {
+        if (
+            managedIdentityRequestParams.claims ||
```

Comment:
does msal js already have a way for app developers to provide this ?

Reply:
yes, but only per request basis. Hence, me adding the claims to the MI request params.